rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/is_hvac_sys_fluid_loop_attached_to_chiller.py

```diff
@@ -0,0 +1,60 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.utils.assertions import getattr_
+from rct229.utils.jsonpath_utils import (
+    find_all,
+    find_exactly_one_with_field_value,
+    find_one_with_field_value,
+)
+
+FLUID_LOOP = schema_enums["FluidLoopOptions"]
+
+
+def is_hvac_sys_fluid_loop_attached_to_chiller(rmi_b, hvac_b_id):
+    """Returns TRUE if the fluid loop associated with the cooling system associated with the HVAC system is attached to a chiller. Returns FALSE if this is not the case.
+
+    Parameters
+    ----------
+    rmi_b : json
+        RMD at RuleSetModelInstance level
+    hvac_b_id : str
+        The HVAC system ID.
+
+    Returns
+    -------
+    bool
+        True: fluid loop associated with the cooling system associated with the HVAC system is attached to a chiller
+        False: otherwise
+    """
+    is_hvac_sys_fluid_loop_attached_to_chiller_flag = False
+    chillers = find_all("$.chillers[*]", rmi_b)
```

Comment:
I wonder if we should try to use _id_ in the variable name when it refers to an id rather than the actual object.
In this case `chiller_ids` might make it easier for us to keep the distinction straight.

Reply:
I will address the variable names for ids. This line is getting the actual `chillers`, not `chiller_ids`